### ISSUE TITLE
"metadata_val.csv" is not referenced by default config.json files but is referenced in docs

### ISSUE THREAD

**Issue by GuyPaddock** [mozilla/TTS#665]
The README and tutorials in this project mention that when training a custom voice you want to split the data set into `metadata_train.csv` and `metadata_val.csv` files before running the training process with `./TTS/tts/configs/config.json`. But, that JSON file does not reference `metadata_train.csv` or `metadata_val.csv` at all. Either the docs should be updated to recommend updating the relevant portion of `config.json` to reference these files, OR the default configs should expect them.

**Comment by erogol** [comment]
If you don't provide a validation file, it automatically takes a portion of the training data as the validation set, if this is what you mean.

**Comment by GuyPaddock** [comment]
@erogol No, what I mean is that tutorials and README encourage you to create a `metadata_train.csv` and `metadata_val.csv` file as part of the training process but the instructions don't mention that you need to customize `config.json` to use those files.
